tests/hardhat/Swap/swapTest.ts

```diff
@@ -440,6 +463,16 @@ describe("Swap Contract", () => {
       );
     });
 
+    it("swap tokenA -> full debt of tokenB", async () => {
+      const deadline = await getValidDeadline();
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      await expect(
+        swapRouter
+          .connect(user)
+          .swapTokensForFullTokenDebtAndRepay(vToken.address, SWAP_AMOUNT, [tokenA.address, tokenB.address], deadline),
+      );
```

Comment:
Shouldn't we expect some result in this case? If we don't expect a result, this seems to be equivalent to `await swapRouter.connect(user).swapTokensForFullTokenDebtAndRepay(...)` 🤔

I guess this is the reason eslint reports a floating promise here

Reply:
In other tests also we are just verifying that they are not throwing any error, as we are using smock's fake contracts so we are unable to emit events.